src/videojs-http-streaming.js

```diff
@@ -172,16 +207,16 @@ const emeKeySystems = (keySystemOptions, videoPlaylist, audioPlaylist) => {
 };
 
 const setupEmeOptions = (hlsHandler) => {
-  if (hlsHandler.options_.sourceType !== 'dash') {
-    return;
-  }
+  const mainSegmentLoader = hlsHandler.masterPlaylistController_.mainSegmentLoader_;
+  const audioSegmentLoader = hlsHandler.masterPlaylistController_.audioSegmentLoader_;
```

Comment:
Does this work for both muxed and umuxed content?

Reply:
What do you mean by "this"?

Reply:
The current changes really. Specifically, with muxed audio, we don't have a separate audio segment loader. We should make sure that this works in either case, or at least, doesn't break existing streams.

Reply:
Oh, I guess it's unlikely to break existing streams because dash is always unmuxed.

Reply:
If the content is muxed, then `audioSegmentLoader` is `null` and `emeKeySystems()` will pull the relevant audio/video codec info from the `mainSegmentLoader.mimeType_` and format it into separate audio/video mime types accordingly. If it is unmuxed, then the audio/video codec info has already separated out and formatted by [`configureLoaderMimeTypes_()`](https://github.com/videojs/http-streaming/blob/1.x/src/master-playlist-controller.js#L1127-L1130) into the exact format we need to setup eme.